fms2_io/include/get_global_attribute.inc

```diff
@@ -32,8 +32,15 @@ subroutine get_global_attribute_0d(fileobj, &
                                              !! The broadcast will be done
                                              !! by default.
     integer :: err
+    character(len=1024), dimension(1) :: charbuf !< 1D Character buffer
```

Comment:
Is there a reason charbuf needs to be defined as an array of length '1'?

Reply:
I'm not sure there is a scalar string broadcast:

https://github.com/NOAA-GFDL/FMS/blob/94afa26669c1d1659f670cb739c0e9b4e51e7ece/mpp/include/mpp_comm_mpi.inc#L415